### PR TITLE
Features/data dog dashboard

### PR DESCRIPTION
**Pull request by kyle-ssg**
Thanks for submitting a PR! Please check the boxes below:

- [x] I have run [`pre-commit`](https://docs.flagsmith.com/deployment/locally-api#pre-commit) to check linting
- [x] I have filled in the "Changes" section below?
- [x] I have filled in the "How did you test this code" section below?

## Changes

image

https://user-images.githubusercontent.com/8608314/207095553-ccd4003f-d792-43bc-9ba6-fc6a6041616d.mov

### CODE REVIEW

File: frontend/common/stores/feature-list-store.js [hunk 1]
```diff
@@ -37,7 +37,7 @@ const controller = {
                 filterUrl = `&${Utils.toParam(store.filter)}`;
             }
 
-            let featuresEndpoint = `${Project.api}projects/${projectId}/features/?page=${page || 1}&page_size=${PAGE_SIZE}${filterUrl}`;
+            let featuresEndpoint = typeof page === 'string'? page: `${Project.api}projects/${projectId}/features/?page=${page || 1}&page_size=${pageSize||PAGE_SIZE}${filterUrl}`;
```
Comment (kyle-ssg) [inline frontend/common/stores/feature-list-store.js:40]: I realised that paging was broken for features, so people with 1000+ flags can't currently go back to page 1 without refreshing.

File: frontend/web/components/FeatureRow.js [hunk 2]
```diff
@@ -201,29 +199,32 @@ class TheComponent extends Component {
...
+                                                    removeFlag(projectId, projectFlag);
+                                                })}
+                                                className="btn btn--with-icon"
+                                                data-test={`remove-feature-btn-${this.props.index}`}
+                                            >
+                                                <RemoveIcon/>
+                                            </button>
+                                        )}
+                                    >
+                                        {isProtected ? '<span>This feature has tagged as <bold>protected</bold>, <bold>permanent</bold>, <bold>do not delete</bold>, or <bold>read only</bold>. Please remove the tag before attempting to delete this flag.</span>' : 'Remove feature'}
```
Comment (matthewelwell) [inline frontend/web/components/FeatureRow.js]:
```suggestion
                                        {isProtected ? '<span>This feature has been tagged as <bold>protected</bold>, <bold>permanent</bold>, <bold>do not delete</bold>, or <bold>read only</bold>. Please remove the tag before attempting to delete this flag.</span>' : 'Remove feature'}
```

File: frontend/web/components/pages/AuditLogPage.js [hunk 3]
```diff
@@ -1,172 +0,0 @@
-import React, { Component } from 'react';
```
Comment (matthewelwell) [inline frontend/web/components/pages/AuditLogPage.js:1]: I think this file can be completely removed? Seems like we've just truncated it?

File: frontend/web/main.js [hunk 4]
```diff
@@ -91,8 +91,8 @@ ReactDOM.render(<ToastMessages/>, document.getElementById('toast'));
 if (E2E) {
     document.body.classList.add('disable-transitions');
 }
-
-if (!E2E && Project.crispChat) {
+const isWidget = document.location.href.includes("/datadog")
```
Comment (matthewelwell) [inline frontend/web/main.js]: I noticed earlier we had some logic based on /widget, but here we're basing it on /datadog. Is there a reason for this?

**Review by matthewelwell — APPROVED** [review]

**Pull request merged**